packages/react-server/src/ReactFizzServer.js

```diff
@@ -1899,7 +1899,7 @@ function flushCompletedQueues(request: Request): void {
 }
 
 export function startWork(request: Request): void {
-  scheduleWork(() => performWork(request));
+  scheduleWork(request.destination, () => performWork(request));
```

Comment:
While it might make sense to want to schedule based on the destination, the assumption of this API implies a lot more of a global scheduler like postTask. I worry that this might take us down the wrong assumptions.

For sync I/O, it'll also be required that it's an instance (or at least thread) per request.

Reply:
Eh, just some flexibility. I can revert it and make it use a global instead, that wouldn't be a problem.